SoftLayer/testing/xmlrpc.py

```diff
@@ -66,6 +66,15 @@ def do_POST(self):
             except UnicodeDecodeError:
                 self.wfile.write(response_body)
 
+        except NotImplementedError as ex:
```

Comment:
Also catch `NameError` here. As that will handle any fixtures that exist, but don't have the method defined in them.

Like this:
```
except (NotImplementedError, NameError) as ex:
```

Reply:
I have added it.